.github/workflows/build.yml

```
@@ -53,7 +53,8 @@ jobs:
     - name: PyLint
       run: |
         pylint $( find . -path './adafruit*.py' )
-        ([[ ! -d "examples" ]] || pylint --disable=missing-docstring,invalid-name,bad-whitespace $( find . -path "./examples/*.py" ))
+        ([[ ! -d "examples" ]] || pylint --disable=missing-docstring,invalid-name,bad-whitespace $( find . -name "advanced_examples" -prune -false -o -path "./examples/*.py" ))
+        ([[ ! -d "examples/advanced_examples" ]] || pylint --disable=missing-docstring,invalid-name,bad-whitespace $( find . -path "./examples/advanced_examples/*.py" ))
```

Please try the following instead of creating a new line to check the examples subfolder. It worked for me locally, but I would appreciate it if you could test it as well.
```suggestion
        ([[ ! -d "examples" ]] || pylint --disable=missing-docstring,invalid-name,bad-whitespace $( find examples -name \*.py ))
```

---

I tested that change in https://github.com/AlexColello/Adafruit_CircuitPython_CLUE/commit/3f957cb575a597ed91e8ecd299ae6935de080775, but it causes the same error as in https://github.com/adafruit/Adafruit_CircuitPython_CLUE/pull/39/commits/91674a178f699aa5bc7198a6afca3a55e5091318. I checked the find command and both this and the original produce the same results, so I don't think that is the issue.